tools/autograd/derivatives.yaml

```diff
@@ -514,6 +514,10 @@
 - name: hardswish(Tensor self) -> Tensor
   self: hardswish_backward(grad, self)
 
+- name: hypot(Tensor self, Tensor other) -> Tensor
+  self: grad * self / sqrt(self * self + other * other)
```

Comment:
Great Job! 
Just a passing suggestion. 
You can probably use `result` instead of recomputing with `sqrt(self * self + other * other)`

Example: 
https://github.com/pytorch/pytorch/blob/be19bda57e9f6cdce86bb81309dacf3cea1c48f3/tools/autograd/derivatives.yaml#L424-L428